soc/arm/st_stm32/stm32f3/Kconfig.defconfig.stm32f303xe

```diff
@@ -0,0 +1,27 @@
+# ST Microelectronics STM32F303XC MCU
+
+# Copyright (c) 2020 Paul M. Bendixen
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32F303XE
+
+config SOC
+	default "stm32f303xe"
+
+config NUM_IRQS
+	default 82
```

Comment:
85

Reply:
Shouldn't it be 84 as that is the highest number?
Just following https://github.com/zephyrproject-rtos/zephyr/commit/ae2742dc6bf3208a308dd047bec3d9c127492966
Couldn't find better documentation for the value

Reply:
Well, number should actually be highest IRQ number +1 as is starts from 0.

Reply:
Ok